arrays/test.js

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
This is interesting. Why the inclusion of the balancedBrackets() function here?

Reply:
Oops, I had issues with one of my brackets so I copied over since I have it the bracket pair colorizer in vs code. I thought I had deleted it before committing XD.